documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -245,6 +263,20 @@ Note that the Domain custom resource definition is shared.
 Do not delete the CRD if there are other operators in the same cluster
 or you have running domain resources.
 
+Beginning with Operator version 4.0, uninstalling an operator also removes the conversion webhook
+ deployment and its associated resources by default.
+Therefore, if you have multiple operators running, then, by default, an uninstall 
+of one operator will affect the other operators. The uninstall will not delete the conversion definition 
+in the domain CRD so you will be unable to create domains using `weblogic.oracle/v8` schema.
```

Comment:
How about simply:

The uninstall will not delete the domain CRD's reference to the webhook, so you will be unable to create new domains or alter existing domains.